moveit_ros/moveit_servo/include/moveit_servo/servo_parameters.h

```diff
@@ -53,53 +54,76 @@ struct ServoParameters
-  std::string joint_command_in_topic;
-  std::string robot_link_command_frame;
-  std::string command_in_type;
-  double linear_scale;
-  double rotational_scale;
-  double joint_scale;
+  std::string cartesian_command_in_topic = "~/delta_twist_cmds";
+  std::string joint_command_in_topic = "~/delta_joint_cmds";
+  std::string robot_link_command_frame = "panda_link0";
+  std::string command_in_type = "unitless";
+  double linear_scale = 0.4;
+  double rotational_scale = 0.8;
+  double joint_scale = 0.5;
   // Properties of outgoing commands
-  std::string command_out_topic;
-  double publish_period;
-  std::string command_out_type;
-  bool publish_joint_positions;
-  bool publish_joint_velocities;
-  bool publish_joint_accelerations;
+  std::string command_out_topic = "/panda_arm_controller/joint_trajectory";
+  double publish_period = 0.034;
+  std::string command_out_type = "trajectory_msgs/JointTrajectory";
+  bool publish_joint_positions = true;
+  bool publish_joint_velocities = true;
+  bool publish_joint_accelerations = false;
   // Plugins for smoothing outgoing commands
-  std::string joint_topic;
-  std::string smoothing_filter_plugin_name;
+  std::string joint_topic = "/joint_states";
+  std::string smoothing_filter_plugin_name = "online_signal_smoothing::ButterworthFilterPlugin";
   // MoveIt properties
-  std::string move_group_name;
-  std::string planning_frame;
-  std::string ee_frame_name;
-  bool is_primary_planning_scene_monitor;
-  std::string monitored_planning_scene_topic;
+  std::string move_group_name = "panda_arm";
+  std::string planning_frame = "panda_link0";
+  std::string ee_frame_name = "panda_link8";
+  bool is_primary_planning_scene_monitor = true;
+  std::string monitored_planning_scene_topic =
+      planning_scene_monitor::PlanningSceneMonitor::DEFAULT_PLANNING_SCENE_TOPIC;
   // Stopping behaviour
-  double incoming_command_timeout;
-  int num_outgoing_halt_msgs_to_publish;
-  bool halt_all_joints_in_joint_mode;
-  bool halt_all_joints_in_cartesian_mode;
+  double incoming_command_timeout = 0.1;
+  int num_outgoing_halt_msgs_to_publish = 4;
+  bool halt_all_joints_in_joint_mode = true;
+  bool halt_all_joints_in_cartesian_mode = true;
   // Configure handling of singularities and joint limits
-  double lower_singularity_threshold;
-  double hard_stop_singularity_threshold;
-  double joint_limit_margin;
-  bool low_latency_mode;
+  double lower_singularity_threshold = 17.0;
+  double hard_stop_singularity_threshold = 30.0;
+  double joint_limit_margin = 0.1;
+  bool low_latency_mode = false;
   // Collision checking
-  bool check_collisions;
-  double collision_check_rate;
-  double self_collision_proximity_threshold;
-  double scene_collision_proximity_threshold;
+  bool check_collisions = true;
+  double collision_check_rate = 10.0;
+  double self_collision_proximity_threshold = 0.01;
+  double scene_collision_proximity_threshold = 0.02;
+  // Pose Tracking parameters
```

Comment:
Please don't put the pose tracking parameters in here.  Please create a separate pose tracking parameters object that uses the same pattern and declares/gets these.